superset/dashboards/api.py

```diff
@@ -160,19 +160,25 @@ def post(self) -> Response:
             500:
               $ref: '#/components/responses/500'
         """
+        self.incr_stats("init", self.post.__name__)
```

Comment:
Can we dry this up? Why not just increment inside of `response_xxx` and set the method name in `@expose`?

Reply:
Good idea! (this is making things a bit uglier)